experimental/PropertyDDS/packages/property-proxy/jest.config.cjs

```diff
@@ -19,10 +19,14 @@ module.exports = {
 	transform: {
 		"^.+\\.ts$": "ts-jest",
 	},
+	moduleNameMapper: {
+		// Remove explicit .js from local paths to allow jest to find the .ts* files
+		"^(\\.{1,2}/.*)\\.js$": "$1",
+	},
 	globals: {
 		"ts-jest": {
 			tsconfig: "src/test/tsconfig.json",
 		},
 	},
-	testPathIgnorePatterns: ["/node_modules/", "dist"],
+	testPathIgnorePatterns: ["/node_modules/", "dist", "lib"],
```

Comment:
Maybe. Is there a default match for jest tests? I don't see anything in this config.

Reply:
According to [the docs](https://jestjs.io/docs/configuration#testpathignorepatterns-arraystring), the default is `["/node_modules/"]`

Reply:
So assuming jest is supposed to test the .ts files, not the built JS files, then adding lib here is correct.